lib/cipherstash/protect/model/crud.rb

```diff
@@ -75,28 +75,21 @@ def match(args = {})
               end
             end
 
-            query = ""
+            query = []
```

Comment:
oh, oops, i'd missed this bit entirely in the last review, sorry 😅

Reply:
all good! thanks again for the suggestion to improve the code. 😄